.github/workflows/cla.yml

```
@@ -0,0 +1,115 @@
+        github.event.issue.state == 'open' &&
+        (
+          github.event.comment.body == 'recheck' ||
+          github.event.comment.body == 'I have read the CLA Document and I hereby sign the CLA'
+        )
+      )
+    runs-on: ubuntu-latest
+    # Signature updates share one file and must not race. Keep all pending
+    # events instead of replacing the previous pending run.
+    concurrency:
+      group: cla-signatures
+      queue: max
```

actionlint (1.7.12)

[error] unexpected key "queue" for "concurrency" section. expected one of "cancel-in-progress", "group"

(syntax-check)